i18n/en.pot

```diff
@@ -59,6 +59,12 @@ msgstr ""
 msgid "Delete"
 msgstr ""
 
+msgid "Delete Instance?"
+msgstr ""
+
+msgid "Are you sure you want to delete this instance?"
+msgstr ""
+
```

Comment:
Are those two literals missing for es.po?

Reply:
there is no es.po file in this project yet, I'll generate one on a separate PR

Reply:
No need to generate any es.po. They don't want multilingual as this app is planned to be used by English speakers